tableau/client.go

```diff
@@ -112,6 +115,23 @@ func (c *Client) signIn(personalAccessTokenName, personalAccessTokenSecret, site
 	return nil
 }
 
+// do makes an HTTP request and return body raw
+func (c *Client) doRaw(ctx context.Context, req *http.Request) (string, error) {
```

Comment:
Tableau API can be tricky sometimes. :) But I don't think we need to add a new method for this.
If an API endpoint returns a CSV data we can use it with "string" type to return from our client.